releases/release-7.4.0.md

```diff
@@ -0,0 +1,186 @@
+---
+title: TiDB 7.4.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.4.0.
+---
+
+# TiDB 7.4.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.4.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.4.0#version-list)
+
+7.4.0 introduces the following key features and improvements:
+
+<!-- key feature placeholder-->
+
+## Feature details
+
+### Performance
+
+- note 1
+
+- note 2
+
+### Reliability
+
+- note 1
+
+- note 2
+
+### SQL
+
+- note 1
+
+- note 2
+
+### DB operations
+
+- note 1
+
+- note 2
+
+### Observability
+
+- note 1
+
+- note 2
+
+### Data migration
+
+- note 1
+
+- note 2
+
```

Comment:
```suggestion


TiCDC improves large message handling with claim check pattern
Before v7.4.0, TiCDC was not able to send large messages downstream to the Kafka broker that would have exceeded Kafkas maximum message size. Starting from v7.4.0, TiCDC allows for users to configure an external storage location where these large messages can be stored. Instead of the large message, a reference (or claim check) is used in the message and sent downstream to the Kafka broker. Message consumers can then retrieve the message content from the external storage location when needed. See[documentation for further details]

```